CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to this project will be documented in this file.
 -   Allow interfaces in classes.
 -   Python support for a simple class.
 -   #1430 : Added conjugate support to booleans.
+-   #1455 : Improve the class Constructor Call
```

Comment:
I think this should either go in the "Changes" section or be removed. The aim of the CHANGELOG is to help people find when changes were made. This comment is very non-specific so I am not sure it is particularly useful. Maybe : 
```
Make `ConstructorCall` inherit from `FunctionCall`.
```

Reply:
Fixed